app/controllers/users_controller.rb

```diff
@@ -11,7 +11,7 @@ def show
 
   def update
     current_user.update_and_notify! user_params
-    redirect_to root_path, notice: I18n.t(:user_data_updated)
+    redirect_to user_path, notice: I18n.t(:user_data_updated)
```

Comment:
I changed this so you wouldn't be sent back to root after editing your profile.

However, we might want that to have the old behavior after the very first profile edition, the mandatory one on the first sign in. Should we take that into... account?

Reply:
Yep. I'll merge this one and try to address this issue in particular in gamification branch